system-tests/lib/performance-reporter.js

```diff
@@ -0,0 +1,81 @@
+const path = require('path')
+const chalk = require('chalk')
+const Libhoney = require('libhoney')
+
+const pkg = require('@packages/root')
+const ciProvider = require('@packages/server/lib/util/ci_provider')
+const { commitInfo } = require('@cypress/commit-info')
+
+class StatsdReporter {
+  constructor (runner) {
+    if (!process.env.HONEYCOMB_API_KEY) {
+      return
+    }
+
+    console.log(chalk.green('Reporting to honeycomb'))
+
+    let branch; let commitSha
+
+    this.honey = new Libhoney({
+      dataset: 'systemtest-performance',
+      writeKey: process.env.HONEYCOMB_API_KEY,
+    })
+
+    commitInfo().then((commitInformation) => {
+      const ciInformation = ciProvider.commitParams() || {}
+
+      this.branch = commitInformation.branch || ciInformation.branch
+      this.commitSha = commitInformation.sha || ciInformation.sha
+    })
+
+    runner.on('test', (test) => {
+      test.wallclockStart = Date.now()
+    })
+
+    runner.on('test end', (test, done) => {
```

Comment:
Do we need to call `done` here? I feel like that's usually how the mocha apis work 🤔